localstack/services/install.py

```diff
@@ -313,7 +312,7 @@ def install_kinesis_mock():
 
 
 def install_local_kms():
-    local_arch = get_os()
+    local_arch = f"{platform.system().lower()}-{get_arch()}"
     binary_path = INSTALL_PATH_KMS_BINARY_PATTERN.replace("<arch>", local_arch)
     if not os.path.exists(binary_path):
         log_install_msg("KMS")
```

Comment:
Since you implemented the multiarch support for local-kms, could you please remove the according TODO in line 320?

Reply:
Hi @alexrashed , good catch. Thx.